native/src/utils/AppSettings.ts

```diff
@@ -36,22 +35,28 @@ class AppSettings {
   }
 
   loadSettings = async (): Promise<SettingsType> => {
-    const settingsKeys = Object.keys(defaultSettings)
+    const settingsKeys = Object.keys(defaultSettings) as [keyof SettingsType]
     const settingsArray = await this.asyncStorage.multiGet(settingsKeys)
-    return mapValues(fromPairs(settingsArray), (value, key) => {
+    const settings = fromPairs(settingsArray) as Record<keyof SettingsType, string | null>
+    return mapValues(settings, (value: string | null, key) => {
```

Comment:
This does not work right?
```suggestion
    return mapValues(settings, (value: string | null, key: SettingsType) => {
```

Reply:
Nope. As map Values explicitly infers key as string